#### I developed a list of approximately 170 proteins that were differentially expressed in at least 3 eelgrass sites OR at least 4 eelgrass sites, annotated with Uniprot Accession Numbers, and then added a synopsis of the function (found via a combination of Uniprot, NCBI, (coughWikipediacough), articles, etc., defined my understanding of their biological categories, and then rated them on my interest level - 5 highest, 1 lowest.  This annotated diff. expressed protein list is found here: [2017-07-04_Proteins-of-interest->2x-annotated.xlsx](https://github.com/laurahspencer/Geoduck-DNR/blob/master/Analyses/2017-June_Analyses/2017-07-04_Proteins-of-interest-%3E2x-annotated.xlsx), and here is the list of all annotated proteins found in all my samples: [All Annotated Proteins](https://github.com/laurahspencer/Geoduck-DNR/blob/master/Analyses/2017-June_Analyses/2017-06-30_All-Annotated-Proteins.tab)

#### From the diff. expressed protein list, my understanding of environmental factors, and lit review, these are the categories I would like to include, and some potential proteins to target:
* **Starvation** 
    * Bax inhibitor 1 (BI-1) (Testis-enhanced gene transcript protein (Transmembrane BAX inhibitor motif-containing protein 6); P5506, Suppressor of apoptosis. Modulates unfolded protein response signaling. Modulates ER calcium homeostasis by acting as a calcium-leak channel. Negatively regulates autophagy and autophagosome formation, especially during periods of nutrient deprivation, and reduces cell survival during starvation
    * Acetyl-coenzyme A synthetase 2-like, mitochondrial (EC 6.2.1.1) (Acetate--CoA ligase 2) (Acetyl-CoA synthetase 2) (AceCS2) (Acyl-CoA synthetase short-chain family member 1); Q99NB1; Important for maintaining normal body temperature during fasting and for energy homeostasis. Essential for energy expenditure under ketogenic conditions. Converts acetate to acetyl-CoA so that it can be used for oxidation through the tricarboxylic cycle to produce ATP and CO2       
* **pH regulation**
    * Sodium-driven chloride bicarbonate exchanger (Solute carrier family 4 member 10); Q5DTL9; Electrogenic sodium/bicarbonate cotransporter in exchange for intracellular chloride. Plays an important role in regulating intracellular pH       
* **Response to high or low oxygen**  
    * Hemoglobin-3 (Hemoglobin III) (Hb III); P41262; "This molluscan globin lacks one of the heme-binding histidine residues found in most other globins."       
    * Cytochrome b5 domain-containing protein 1; heme axial ligand    Q567I9    
    * Globin; P02207; The globins are a superfamily of heme-containing globular proteins, involved in binding and/or transporting oxygen. These proteins all incorporate the globin fold, a series of eight alpha helical segments. Two prominent members include myoglobin and hemoglobin."      
* **Xenobiotics**
    * Sulfotransferase 1C4 (ST1C4) (EC 2.8.2.-) (Sulfotransferase 1C2) (SULT1C#2); O75897; Sulfotransferase that utilizes 3'-phospho-5'-adenylyl sulfate (PAPS) as sulfonate donor to catalyze the sulfate conjugation of drugs, xenobiotic compounds, hormones, and neurotransmitters. May be involved in the activation of carcinogenic hydroxylamines. Shows activity towards p-nitrophenol and N-hydroxy-2-acetylamino-fluorene (N-OH-2AAF). My be involved in ethanol breakdown - see if this shows up in Yaaminiâ€™s 
* **Immune system response**
    * Achacin; P35903; Antibacterial glycoprotein. (interesting paper: http://www.sciencedirect.com/science/article/pii/S0014579302036086)
* **Metabolism**
    * AMP deaminase 2 (EC 3.5.4.6) (AMP deaminase isoform L); Q01433
    * AMP deaminase (EC 3.5.4.6) (Myoadenylate deaminase); P15274
    * Mannose-6-phosphate isomerase (EC 5.3.1.8) (Phosphohexomutase) (Phosphomannose isomerase) (PMI); Involved in the conversion of glucose to GDP-L-fucose, which can be converted to L-fucose, a capsular polysaccharide    Q83KZ1
    *  Betaine--homocysteine S-methyltransferase 1 (EC 2.1.1.5); " Involved in the regulation of homocysteine metabolism. Converts betaine and homocysteine to dimethylglycine and methionine, respectively. This reaction is also required for the irreversible oxidation of choline."    O35490       
* **ROS**
    * Thioredoxin, mitochondrial (MTRX) (Mt-Trx) (Thioredoxin-2); P97615; Important for the control of mitochondrial reactive oxygen species homeostasis, apoptosis regulation and cell viability. Possesses a dithiol-reducing activity.       
    * L-ascorbate oxidase (ASO) (Ascorbase) (EC 1.10.3.3); P24792; May be involved in a redox system involving ascorbic acid.
    * Glutathione synthetase (GSH synthetase) (GSH-S) (EC 6.3.2.3) (Glutathione synthase); P48637; Glutathione synthetase (GSS) (EC 6.3.2.3) is the second enzyme in the glutathione (GSH) biosynthesis pathway. It catalyses the condensation of gamma-glutamylcysteine and glycine, to form glutathione.[2] Glutathione synthetase is also a potent antioxidant. Glutathione (GSH) is an important antioxidant in plants, animals, fungi, and some bacteria and archaea. Glutathione is capable of preventing damage to important cellular components caused by reactive oxygen species such as free radicals, peroxides, lipid peroxides, and heavy metals.
* **Regulation of alternative splicing**
    * RNA binding protein fox-1 homolog 1, 2 or 3; (A6NFN3, Q5NVN8, A6QPR6)
    * U1 small nuclear ribonucleoprotein C (U1 snRNP C) (U1-C) (U1C); Component of the spliceosomal U1 snRNP, which is essential for recognition of the pre-mRNA 5' splice-site and the subsequent assembly of the spliceosome. SNRPC/U1-C is directly involved in initial 5' splice-site recognition for both constitutive and regulated alternative splicing. The interaction with the 5' splice-site seems to precede base-pairing between the pre-mRNA and the U1 snRNA. Stimulates commitment or early (E) complex formation by stabilizing the base pairing of the 5' end of the U1 snRNA and the 5' splice-site region. (from homo sapien gene, P09234    C3Z1P5    
* **Epigenetic regulation**
    * Histone-lysine N-methyltransferase SETD2 (EC 2.1.1.43) (HIF-1) (Huntingtin yeast partner B) (Huntingtin-interacting protein 1) (HIP-1) (Huntingtin-interacting protein B) (Lysine N-methyltransferase 3A) (SET domain-containing protein 2) (hSET2) (p231HBP); Q9BYW2; Histone methyltransferase that specifically trimethylates 'Lys-36' of histone H3 (H3K36me3) using dimethylated 'Lys-36' (H3K36me2) as substrate. Represents the main enzyme generating H3K36me3, a specific tag for epigenetic transcriptional activation    
    * Lysine-specific demethylase 8 (EC 1.14.11.27) (JmjC domain-containing protein 5) (Jumonji domain-containing protein 5); B5XF11; Histone demethylase required for G2/M phase cell cycle progression. Specifically demethylates dimethylated 'Lys-36' (H3K36me2) of histone H3, an epigenetic repressive mark, thereby acting as a transcription activator    
    * Proteasome activator complex subunit 4 (Proteasome activator PA200); Associated component of the proteasome that specifically recognizes acetylated histones and promotes ATP- and ubiquitin-independent degradation of core histones during DNA damage response. binds to acetylated histones and promotes degradation of histones; Q6NRP2    
* **Male/female determinant**
    * ? _none found in diff expressed proteins_
* **Shell formation**
    * Gigasin-3a, P86786, Component of the organic matrix of calcified shell layers
* **Ciliary activity**
    * Dynein heavy chain 12, axonemal (Axonemal beta dynein heavy chain 12) (Axonemal dynein heavy chain 12-like protein) (Axonemal dynein heavy chain 7-like protein) (Ciliary dynein heavy chain 12) (Dynein heavy chain 7-like, axonemal) (Dynein heavy chain domain-containing protein 2); Q6ZR08; Force generating protein of respiratory cilia. Produces force towards the minus ends of microtubules. Dynein has ATPase activity; the force-producing power stroke is thought to occur on release of ADP. Involved in sperm motility; implicated in sperm flagellar assembly        
    * WD repeat-containing protein 34; Critical for ciliary functions, essential to normal development and survival, most probably as a previously unrecognized component of the mammalian dynein-motor-based intraflagellar transport (IFT) machinery. Acts as a negative regulator of the Toll-like and IL-1R receptor signaling pathways. Inhibits the MAP3K7-induced NF-kappa-B activation pathway. Inhibits MAP3K7 phosphorylation at 'Thr-184' and 'Thr-187' upon Il-1 beta stimulation (By similarity).    Q5U4F6    
* Categories diff. regulated in Thompson et al. 2015
    * **Energy production**  
        * ?? 
    * **Stress responses**  
        * Heat shock 70 kDa protein 12B; Regulation of HSF1-mediated heat shock response.     Q96MM6    
        * Universal stress protein MJ0531 (USP MJ0531); Archaea gene/protein; belongs to the universal stress protein A family    Q57951    
        * Corticotropin-releasing factor-binding protein (CRF-BP) (CRF-binding protein) (Corticotropin-releasing hormone-binding protein) (CRH-BP); Q91653; Corticotropin-releasing factor binding protein (CRH-BP) is thought to play an inhibitory role in which it binds CRH and other CRH-like ligands and prevents the activation of CRH receptors. There is however evidence that CRH-BP may also exhibit diverse extra and intracellular roles in a cell specific fashion and at specific times in development       
        * HIG1 domain family member 1A, mitochondrial; Q5NVQ1; Proposed subunit of cytochrome c oxidase (COX, complex IV), which is the terminal component of the mitochondrial respiratory chain that catalyzes the reduction of oxygen to water. May play a role in the assembly of respiratory supercomplexes. Electron transport, Respiratory chain, Stress response, Transport; negative regulation of apoptosis;oxidation-reduction; response to hypoxia"       
    * ** Cytoskeleton**  
        * Talin-2; As a major component of focal adhesion plaques that links integrin to the actin cytoskeleton, may play an important role in cell adhesion. Recruits PIP5K1C to focal adhesion plaques and strongly activates its kinase activity (By similarity).By similarity    Q71LX4    
        *  Nesprin-1 (Enaptin) (Myocyte nuclear envelope protein 1) (Myne-1) (Nuclear envelope spectrin repeat protein 1) (Synaptic nuclear envelope protein 1) (Syne-1); Multi-isomeric modular protein which forms a linking network between organelles and the actin cytoskeleton to maintain the subcellular spatial organization. As a component of the LINC (LInker of Nucleoskeleton and Cytoskeleton) complex involved in the connection between the nuclear lamina and the cytoskeleton. The nucleocytoplasmic interactions established by the LINC complex play an important role in the transmission of mechanical forces across the nuclear envelope and in nuclear movement and positioning. May be involved in nucleus-centrososme attachment. During interkinetic nuclear migration (INM) at G2 phase and nuclear migration in neural progenitors its LINC complex association with SUN1/2 and probably association with cytoplasmic dynein-dynactin motor complexes functions to pull the nucleus toward the centrosome; SYNE1 and SYNE2 seem to act redundantly in cerebellum, midbrain, brain stem, and other brain regions except cerebral cortex and hippocampus. Required for centrosome migration to the apical cell surface during early ciliogenesis. May be involved in nuclear remodeling during sperm head formation in spermatogenenis; a probable SUN3:SYNE1/KASH1 LINC complex may tether spermatid nuclei to posterior cytoskeletal structures such as the manchette.    Q6ZWR6   
    * **Protein synthesis**  
        *  Peptidyl-prolyl cis-trans isomerase FKBP14 (PPIase FKBP14) (EC 5.2.1.8) (FK506-binding protein 14) (FKBP-14) (Rotamase); PPIases accelerate the folding of proteins during protein synthesis.    Q5R941   
        * Probable rRNA-processing protein EBP2; "Protein involved in the synthesis of ribosomes."    Q9D903    
        * Eukaryotic translation initiation factor 4E (eIF-4E) (eIF4E) (eIF-4F 25 kDa subunit) (mRNA cap-binding protein); "Recognizes and binds the 7-methylguanosine-containing mRNA cap during an early step in the initiation of protein synthesis and facilitates ribosome binding by inducing the unwinding of the mRNAs secondary structures."    O77210    
        * Methionine--tRNA ligase, cytoplasmic (EC 6.1.1.10) (Methionyl-tRNA synthetase) (MetRS); "Charges" tRNA for amino acid binding: ATP + L-methionine + tRNA(Met) = AMP + diphosphate + L-methionyl-tRNA(Met).    Q6PF21    
    * Cell signaling
        * ???
* **Miscelleneous**  
    * Probable 4-coumarate--CoA ligase 3 (4CL 3) (Os4CL3) (EC 6.2.1.12) (4-coumaroyl-CoA synthase 3); Q6ETN3; phenylpropanoid metabolic process, the process of chemical reactions and pathways involving aromatic derivatives of trans-cinnamic acid. "Flavonol Cinnamoyl COA reductase is an enzyme that contributes in the phenylpropanoid biosyntheis in plants and is enhanced for acclimatization processes at different levels of photosynthetic photon flux; no information is available about its role in animals, we can hypothesize that a homologous function to acclimatization to various levels of light can be found in P. margaritifera as the over-expression of this gene is correlated with the presence of a black mantle
    * Murinoglobulin-2; Protease inhibitor    Q6IE52          
    * Transmembrane protein 256 homolog; integral component of membrane    A4K526      
    * Sodium/potassium/calcium exchanger 2, 3 or 4
    * Calpain clp-1 (EC 3.4.22.-); Calcium-regulated protease involved in necrotic cell death    P34308    
